apps/src/lab2/views/components/InstructionsPanel.tsx

```diff
@@ -65,10 +71,12 @@ const InstructionsPanel: React.FunctionComponent<InstructionsPanelProps> = ({
   canShowNextButton,
   hasNextLevel,
   useSecondaryFinishButton,
-  onContinueOrFinish,
```

Comment:
should this be removed from the props interface (may depend on the answer to my question about whether or not we want to pass in all redux values).

Reply:
Good call, it's not used regardless so I'll remove